packages/design-tokens/src/tokens/functional/components/bento/colors.json

```
@@ -0,0 +1,12 @@
+{
+  "brand": {
+    "Bento": {
+      "background": {
```

```suggestion
      "bgColor": {
```

Just getting ahead of token name changes coming up, might as well start using it for new tokens

---

Ok awesome!